### PR TITLE
Abstracting some partials and increasing their reusability

### PR DESCRIPTION
**Pull request by jonas-sk**
This pull request doesn't change the content of the site. It merely tries to generalize some of the existing partials. This is primarily done by the introduction of `two-tiles` to replace `2-up` & `4-up`. Other changes include using the internal templates for Open Graph, Twitter, and Google Analytics instead of the standard code or separating out the button of `text-and-image` so it could be reused elsewhere (in my origin branch, I included it in `two-tiles`as option but I left that out here)

### CODE REVIEW

File: site/content/_index.md [hunk 1]
```diff
@@ -5,16 +5,17 @@ image: /img/home-jumbotron.jpg
 blurb:
     heading: Why Kaldi?
     text: "Kaldi is the coffee store for everyone who believes that great coffee shouldn't just taste good, it should do good too. We source all of our beans directly from small scale sustainable farmers and make sure part of the profits are reinvested in their communities."
-intro:
+products:
     heading: "What we offer"
     text: "Kaldi is the ultimate spot for coffee lovers who want to learn about their java’s origin and support the farmers that grew it. We take coffee production, roasting and brewing seriously and we’re glad to pass that knowledge to anyone."
```
Comment (erquhart) [inline site/content/_index.md]: This is no longer showing on the home page.

File: site/content/_index.md [hunk 2]
```diff
@@ -5,16 +5,17 @@ image: /img/home-jumbotron.jpg
...
-products:
-    - image: img/illustrations-coffee.svg
-      text: "We sell green and roasted coffee beans that are sourced directly from independent farmers and farm cooperatives. We’re proud to offer a variety of coffee beans grown with great care for the environment and local communities. Check our post or contact us directly for current availability."
-    - image: /img/illustrations-coffee-gear.svg
-      text: "We offer a small, but carefully curated selection of brewing gear and tools for every taste and experience level. No matter if you roast your own beans or just bought your first french press, you’ll find a gadget to fall in love with in our shop."
+    buttontext: See all products
+    buttonlink: '/products'
+    tiles:
```
Comment (erquhart) [inline site/content/_index.md]: These images are now much larger.

File: site/content/_index.md [hunk 3]
```diff
@@ -5,16 +5,17 @@ image: /img/home-jumbotron.jpg
...
+products:
     heading: "What we offer"
     text: "Kaldi is the ultimate spot for coffee lovers who want to learn about their java’s origin and support the farmers that grew it. We take coffee production, roasting and brewing seriously and we’re glad to pass that knowledge to anyone."
-products:
-    - image: img/illustrations-coffee.svg
-      text: "We sell green and roasted coffee beans that are sourced directly from independent farmers and farm cooperatives. We’re proud to offer a variety of coffee beans grown with great care for the environment and local communities. Check our post or contact us directly for current availability."
-    - image: /img/illustrations-coffee-gear.svg
-      text: "We offer a small, but carefully curated selection of brewing gear and tools for every taste and experience level. No matter if you roast your own beans or just bought your first french press, you’ll find a gadget to fall in love with in our shop."
+    buttontext: See all products
+    buttonlink: '/products'
```
Comment (erquhart) [inline site/content/_index.md]: Indentation of above two lines is off by one.

**Comment by jonas-sk** [comment]
Admittedly, I didn't thoroughly test this PR after I pulled the changes apart, so I'll check the issues out now

File: site/content/_index.md (hunk 1 above)
Comment (jonas-sk) [reply site/content/_index.md]: It should be fixed now

File: site/content/_index.md (hunk 3 above)
Comment (jonas-sk) [reply site/content/_index.md]: They don't seem off to me and also work. Care to elaborate?

File: site/content/_index.md (hunk 2 above)
Comment (jonas-sk) [reply site/content/_index.md]: Fixed that now :)

**Comment by jonas-sk** [comment]
Please tell me if you require any more changes :smiley:

File: site/content/_index.md (hunk 3 above)
Comment (erquhart) [reply site/content/_index.md]: Weird, not sure what I was seeing before, but I'm not seeing it now - sorry about that!

**Comment by erquhart** [comment]
Thanks for addressing! Just one more issue, the orange images on the home page had their `w-100` class removed, we should keep that class in place. After that, good to merge 👍

**Pull request closed without merging**